cypress/support/pages/signUp.pageObject.js

```diff
@@ -0,0 +1,55 @@
+import PageObject from '../PageObject';
+
+class SignUpPageObject extends PageObject {
+  url = '/#/register';
+
+  get usernameField() {
+    return cy.getByDataCy('username-sign-up');
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'username-sign-up'.